### DISCUSSION TITLE
Paginated Menus allowing more than 1 slot for a slot item.

### DISCUSSION THREAD

**Discussion by WTFBlaze** [BG-Software-LLC/SuperiorSkyblock2#1977] [Pending Feature Requests] [answered]
![image_2024-01-22_143423945](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/35648047/a757fc43-b9f5-45bf-8d3d-981c941d25d1)

When wanting to do custom ui textures with a plugin like Oraxen or ItemAdder you currently cannot allow 1 slot item to take up more than 1 slot for the same item.

I would like to make it where there is a sub config item for paginated menus where it extends the current selected slot item and duplicates it to a second slot.

Example of how this could be done with the config.
```
title: '&lIsland Settings'
previous-menu: true

pattern:
  - '. . . . . @ # #'
  - '. . . . . @ # #'
  - '. . . . . @ # #'
  - '. . . . . < . > .'

slots: '@'
extended-slots: '#'
```

Making it so that it replaces `#` with the currently selected slot item until the next slot item is the goal.

**Comment by OmerBenGera** [comment]
What do you mean by "selected slot"?
I don't understand.

**Reply by WTFBlaze** [reply]
Selected slot would be the current slot item in the list of slot items the gui iterates through. It would just duplicate the same item to the extended spots until the next @ symbol appears in the gui formatting.

**Comment by Lemvs** [comment, marked as answer]
Added in latest dev build, but in a bit different way, because someone may want to "extend" the slot also down, not just to the right. You can see how exactly it works in https://github.com/BG-Software-LLC/SuperiorSkyblock2/pull/3008

@OmerBenGera you can mark this answered and close it.
